spec/API_specification/elementwise_functions.md

```diff
@@ -1136,10 +1136,15 @@ Returns the remainder of division for each element `x1_i` of the input array `x1
 
 Rounds each element `x_i` of the input array `x` to the nearest integer-valued number.
 
+```{note}
+
+The default rounding mode must be round half to even (i.e., rounding ties to the nearest even integer).
```

Comment:
Does it make sense to say "default rounding mode" if there's just one mode? Maybe this note can simply be left out, because the same info is already present in the line you changed under Special Cases?

Reply:
Removed.